### PR TITLE
#233 Update Gitlab integration

### PR DESCRIPTION
**Pull request by uilianries**
This PR includes:

* Updated `conan new` reference to support Gitlab CI.
* Added step-by-step how to create a new Conan project in Gitlab.

### CODE REVIEW

File: reference/commands/new.rst [hunk 1]
```diff
@@ -64,3 +66,8 @@ package testing files.
 
    $ conan new mypackage/1.0@myuser/stable -t -cilg -cio -ciw
 
+- Crete files for gitlab (linux) Continuous integration and set upload conan server:
```
Comment (memsharded) [inline reference/commands/new.rst]: Crete => create

**Comment by uilianries** [comment]
@memsharded Done! I'm using spell checking in my text editor, it warned me about this typo but I didn't see. Anyway, we could add some spell checker to analyzes each new push, should be faster to find typo erros. What do you think?

**Comment by memsharded** [comment]
Sounds good, but what do you mean? Some tool to run in travis? Will it reject PR if errors are encountered? Because the false positives with so many acronyms, brands, etc, are very common.

**Comment by uilianries** [comment]
Yes, run in Travis some tool to check.  
So, we will have a new problem, as you mentioned, false positives. I think in 2 solutions, reserved words, as brands could be add in a suppress file, as done in clang-tidy and cppcheck.  
Or, use `allow_failures` in Travis and just check (user review) when some error was detected.

**Comment by memsharded** [comment]
Personally, it sounds a bit of overhead right now, to setup and run it. The truth is that right now there is no travis CI for the docs, it is directly run in readthedocs. I am mostly fine with a few typos that will eventually be detected and fixed. But just my personal opinion.

**Comment by uilianries** [comment]
Yes, maybe just docs review is enough for now, I agree. I think that to maintain a white list for each new commit or review Travis warnings will be more effort than necessary, maybe will hide real typo-errors. Also, develop this feature will spend some hours to create travis script. Anyway, thanks to discuss the idea.

**Pull request merged**